tangoObjects.py

```diff
@@ -238,8 +248,11 @@ def get(self, block=True, timeout=None):
         else:
             item = self.__db.lpop(self.key)
 
-        # if item:
-        #     item = item[1]
+        if item is None:
+            return None
+
+        if block and item:
```

Comment:
Wondering why it is necessary to perform `item = item[1]` here - the old code had it commented out but seemed to work somehow still.

Also, the shadowing of `item` is admittedly not the best practice in our code, and changing the name to something else would help in readability

Reply:
honestly I'm not sure about this either but it seems like for the blocking version of lpop, it returns both the key of the queue and also the item itself as a tuple, which is why I did item[1]. I'm not sure about why this is the case, tried to lookup the docs but to no avail. If I were to venture a guess, this is because the redis `blpop` actually takes in a list of queue `keys` (queue names I guess) to iterate through -- this key might be there to identify which queue it was popped off from.